masterlist.yaml

```diff
@@ -4892,6 +4892,76 @@ plugins:
     clean:
       - crc: 0x4E4A0351 # v2.0.3
         util: 'SSEEdit v3.2.71 EXPERIMENTAL'
+  - name: 'Distinct Interiors.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/6130/' ]
+    group: *addonsGroup
+    after:
+      - 'EnhancedLightsandFX.esp'
+      - 'ELFX - NoPlayerHomes.esp'
+      - 'Snazzy Furniture and Clutter Overhaul.esp'
+    inc:
+    # CRC for Modular - Core - Always Needed Version
+    # v1.81 B337055B,
+      - name: 'Enhanced Lights and FX'
+        condition: 'not checksum("Distinct Interiors.esp",B337055B) and active("EnhancedLightsandFX.esp") and not active("ELFX - NoPlayerHomes.esp")'
+      - name: 'InnCredible.esp'
+        condition: 'not checksum("Distinct Interiors.esp",B337055B)'
+      - name: 'OpulentThievesGuild.esp'
+        condition: 'not checksum("Distinct Interiors.esp",B337055B)'
+      - 'Snazzy Furniture and Clutter Overhaul.esp'
```

Comment:
Listed on Distinct Interior page as Incompatible due to clipping.
However there appeared to be minor conflicts in xEdit when sorted after Snazzy furniture.
I've marked it as incompatible, but also added a sort after rule.